src/main/java/no/nav/pto/veilarbportefolje/client/ClientConfig.java

```diff
@@ -41,7 +41,7 @@ public AktorClient aktorClient(EnvironmentProperties properties, SystemUserToken
                 properties.getAktorregisterUrl(), APPLICATION_NAME, systemUserTokenProvider::getSystemUserToken
         );
 
-        return new AktorClient(new CachedAktorOppslagClient(aktorOppslagClient), new CachedAktorregisterClient(aktorregisterClient), unleashService);
+        return new AktorClient(new CachedAktorOppslagClient(aktorOppslagClient), aktorregisterClient, unleashService);
```

Comment:
@stianStensli maybe you can help me here...I couldnt find class for caching aktorregiste

Reply:
I will ask Alexander 👍

Reply:
Alexander seems to want us to switch to only using CachedAktorOppslagClient, but that one does not work well with real users in q1....

Reply:
will it be possible to do something like this?
CachedAktorOppslagClient(new AktorregisterHttpClient())

Reply:
Another option: use ToggledAktorOppslagClient